scripts/requirements.txt

```diff
@@ -0,0 +1,2 @@
+colorama
```

Comment:
This looks redundant with the `requirements.txt` in the root of the `nvbench` repo. I'm not sure what consumes these files, but we should probably only have one.

Reply:
removed, I totally missed the requirements.txt in the root directory.